project1/templates/login.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+</head>
+<body>
+<h4>{{msg}}</h4>
+    <div class="container">
+        <div class="row" style="margin-top: 40px">
+            <div class="col-sm-6">
+                <form method="POST">
+                    <div class="form-group">
+                      <label for="email">Username : </label>
+                      <input type="text" name="uname" class="form-control" id="uname">
+                    </div>
+                    <div class="form-group">
+                      <label for="email">Password : </label>
+                      <input type="password" name="pwd" class="form-control" id="pwd">
+                    </div>
+                    <button type="submit" class="btn form-control btn-default">Login</button>
+                </form>
```

Comment:
You have to add the login button to the register page itself. Not create a separate page for it. Add login button to the register form. You have to figure out how to post to different urls from the same form.